index.js

```diff
@@ -120,13 +121,25 @@ fastify.listen(process.env.PORT || 21864, (error, address) => {
 });
 
 let firstLogin = true;
+let lastLogin = null;
+let combos = 1;
 rtmClient.on('authenticated', (data) => {
 	logger.info(`Logged in as ${data.self.name} of team ${data.team.name}`);
+	const now = new Date();
 	if (!firstLogin) {
+		let comboStr = '';
+		if (now - lastLogin <= 2 * 60 * 1000) {
+			combos++;
+			comboStr = `(${combos}コンボ${_.repeat('!', combos)})`
```

Comment:
[nits] lodashを使わなくても書けます
```suggestion
			comboStr = `(${combos}コンボ${'!'.repeat(combos)})`
```